plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -74,7 +74,7 @@ void PrintVerboseLog(sentry_level_t level, const char *message, va_list args, vo
 
 void PrintCrashLog(const sentry_ucontext_t *uctx)
 {
-#if PLATFORM_WINDOWS
+#if PLATFORM_WINDOWS && ENGINE_MAJOR_VERSION == 5
```

Comment:
How about this so if/when 6 lands, it's included automatically? (I assume nothign would work on UE3 anyway)?

```suggestion
#if PLATFORM_WINDOWS && ENGINE_MAJOR_VERSION != 4
```

Reply:
Since we have quite a few places with the version-specific code I've created a separate issue to track this request - #676